core/network/adapters/adapter_errors.hpp

```diff
@@ -17,6 +17,7 @@ namespace kagome::network {
     DATA_SIZE_CORRUPTED,
     PARSE_FAILED,
     UNEXPECTED_VARIANT,
+    CAST_FAILED
```

Comment:
```suggestion
    CAST_FAILED,
```

Reply:
Done. Strange. Which version requires it?